src/components/bookStore.css

```diff
@@ -1,12 +1,62 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@300;700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@300;700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab&display=swap');
+
+body {
+  background-color: #fafafa;
+  min-height: 100vh;
+  box-sizing: border-box;
+  margin: 0;
+}
+
+.Title {
+  font-family: 'Montserrat-Bold', sans-serif;
+  font-weight: bold;
+  font-size: 1.875rem;
+}
+
+.Title,
+.user,
+.oval {
+  color: #0290ff;
+}
+
+.submit,
+.update {
+  background-color: #0290ff;
+  color: #fff;
+  border: none;
+  padding: 0.801rem 1.188rem 0.886rem 1.375rem;
+  border-radius: 3px;
+}
+
+.book,
+.categories,
+.book-title {
+  font-family: 'Montserrat', sans-serif;
+  font-weight: normal;
+  font-size: 0.813rem;
+  opacity: 0.5;
+  letter-spacing: 1.9px;
+  color: #121212;
```

Comment:
- [ ] The book title and category should be distinctive from each other.

- Your design

![Capture web_18-11-2021_133333_localhost](https://user-images.githubusercontent.com/61562626/142416420-6f5a56bc-c717-4c00-b3b7-fa0d221c44db.jpeg)


- Zeplin design

![mmm](https://user-images.githubusercontent.com/61562626/142416047-f084c6e3-62ae-4416-b334-82b598f8610e.jpeg)